### PR TITLE
[BUILD] Jenkins release pipeline.

### PR DESCRIPTION
**Pull request by michal-raska**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1 +1,3 @@
+version=0.3.1
 sparkVersion=2.0.0
```
Comment (jakubhava) [inline gradle.properties]: should be removed, we don't need spark

**Review by jakubhava — APPROVED** [review]
One minor thing, otherwise looks good.

I would also try to publish to public nexus and close the artifact. If the closing is fine, we can be sure that the publishing is correct ( we can drop the closed artifacts)

**Comment by michal-raska** [comment]
@kuba closed in public nexus without problems
image

Content:
image

**Comment by michal-raska** [comment]
What do you think about this versioning schema? For branches other than `master`, the branch name will be appended to project name, so you get artifacts like `ai.h2o:xgboost-predictor-mr-bu​ild-pipeline:0.3.1-SNAPSHOT`. I think it might be useful for testing purposes, etc. WDYT @michalkurka @jakubhava

**Comment by michal-raska** [comment]
@michalkurka tests looks good: http://mr-0xc1:8080/job/h2o-3-pipeline/job/ops-test-xgboost-predictor/2/

**Review by michalkurka — APPROVED** [review]

**Pull request merged**